Project.toml

```diff
@@ -1,21 +1,21 @@
 name = "GraphicalModelLearning"
 uuid = "73787735-dbe8-5e05-9488-dc58b1b075d0"
-repo = "https://github.com/lanl-ansi/GraphicalModelLearning.jl.git"
 authors = ["Marc Vuffray, Carleton Coffrin"]
+repo = "https://github.com/lanl-ansi/GraphicalModelLearning.jl.git"
 version = "0.1.0"
 
 [deps]
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-MathProgBase = "fdba3010-5040-5b88-9595-932c9decdf73"
+MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 Ipopt = "~0.4, ~0.5"
-JuMP = "~0.17, ~0.18"
-MathProgBase = "~0.7"
+JuMP = "~0.19"
+MathOptInterface = "~0.8"
```

Comment:
Because the type `OptimizerFactory` is pulled in from JuMP, I think we can omit this req entirely.

Reply:
Even if we check the `status` with `MOI`?
https://github.com/lanl-ansi/GraphicalModelLearning.jl/pull/25/files#diff-9115caad4ec7a866f3ca7e2d2e0ec77cR329